src/sentry/scim/endpoints/members.py

```diff
@@ -221,7 +224,7 @@ def patch(self, request: Request, organization, member):
                 self._delete_member(request, organization, member)
                 return Response(status=204)
             else:
-                return Response(SCIM_400_INVALID_PATCH, status=400)
+                raise SCIMApiError(detail=SCIM_400_INVALID_PATCH)
```

Comment:
Technically this is a changed response since the old one didn't include the schema